src/__tests__/fromJson.test.js

```diff
@@ -16,21 +16,11 @@ test("manifest-docs.json content", () => {
   const fileContent = `
 {
   "name": "Parliament Site",
-  "version": "1.0.0",
-  "description": "Onboarding docs for Parliament",
-  "author": "DevRel Team",
   "view_type": "mdbook",
```

Comment:
Any reason for keeping this `view_type`? 
There is some logic in the parser, but I am not sure why it is there.